MUXSDKStats/Podfile.lock

```diff
@@ -2,15 +2,21 @@ PODS:
   - Mux-Stats-Core (3.0.3)
 
 DEPENDENCIES:
-  - Mux-Stats-Core (~> 3.0.3)
+  - Mux-Stats-Core (from `https://github.com/muxinc/stats-sdk-objc.git`, branch `nidhik/viewer-device-model`)
```

Comment:
Note to self: remove this and use 'Mux-Stats-Core', '~>3.1.0' when that is released